modules/k3d/k3d.sh

```diff
@@ -82,18 +86,28 @@ delete() {
   fi
 }
 
```

Comment:
Would this be better placed as part of the `tanka` module?

```
make tanka/create-namespaces
```

and would it be useful as a makefile target in its own right?

Reply:
yeah, maybe sits better in the tanka module

was thinking maybe a new "localdev" module might make sense to wrap multiple commands, to make getting started a single command, ie:

```
make localdev/deploy $appname
make localdev/deploy-all
make localdev/delete $appname
make localdev/delete-all
```
deploy(-all) = creates cluster (if not there), creates namespace (if not there), tk applies specified app (or all apps in repo)
delete(-all) = tk deletes specified app or for "all" deletes all the apps and tears down the cluster

Reply:
I'm not sure how well that will pan out in reality - developers are still going to be developing locally around docker-compose (probably even python virtualenvs).

Reply:
i think it's worth trying to encourage (by making as easy as possible) the use of kind/k3d for local dev

otherwise we are duplicating the effort of creating dev environments and they're testing is less like live

Reply:
or maybe just a `make tanka/apply-local` for now